kirby/api/__init__.py

```diff
@@ -0,0 +1,46 @@
+import requests
+
+from .context import ContextManager, ctx
+
+
+class Kirby:
+    def __init__(self, env_signature, session=None):
+        ctx_manager = ContextManager(env_signature)
+        ctx_manager.load()
```

Comment:
Is there a case where you want to define a context manager, but not `load()` it?
If not, then why not put `load()` inside of `__init__`?

Reply:
I didn't see now any case, so I deleted it